tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -3940,6 +3940,37 @@ TEST_F(EnergyPlusFixture, VRFTest_SysCurve)
         ASSERT_TRUE(true);
     }
 
+    // test that when resistive defrost is used and defrost time is 0 the COP is still degradaed due to frost built up
```

Comment:
This comment needs revision and I think it would be good to check the docs to make sure this performance aspect is discussed.